lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -291,6 +291,15 @@ def module_map
         end
       end
 
+      # @return [Array<Pathname>] Paths to include for local pods to assist in development
+      #
+      def developer_files
+        if file = spec_consumer.license[:file]
```

Comment:
I would have used the file accessor's `license` function but that will apply a glob to the directory of the podspec to search for the license, and wanted to avoid false positives for when multiple podspecs are stored in the same directory

Reply:
This is pretty cool, I added this to `pod lib create` back in the day - I think it's also worth putting the README in there, it's a useful reference file and it's pretty common to change it.

<img width="346" alt="screen shot 2017-10-01 at 8 02 31 am" src="https://user-images.githubusercontent.com/49038/31054357-f0b7e27a-a67e-11e7-9150-b40797b40b80.png">

If you were feeling bold, you could also check for:

* A CHANGELOG
* A folder called docs and include the folder reference

Reply:
That's actually where I got the idea from :D 

I had the same thought about adding other files like README, CHANGELOG, etc. but was worried about situations where multiple podspecs are kept in a single folder - they would all have the same readme, changelog, etc. where that might not be the intended result. Do you think that wouldn't be an issue?

Reply:
I *think* it could be an issue because we have deterministic uuids, and including the same file _may_ cause duplicates. Someone more familiar with that code may know. 

In those cases I think it's fine to either not show them, you could see if there is more than 1 podspec in the folder and only add the files then,

Reply:
That's correct regarding deterministic uuids. we've had cases recently in which adding the same file twice would cause warnings. Make sure to try this on a multi platform pod as you might need to namespace the group according to the platform to avoid this.